mock/imagesSchema.js

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
We could try avj on separate branch, remove it from this branch

Reply:
ok